src/game/engine/scenario.h

```diff
@@ -97,6 +97,8 @@ class ScenarioClass
     BOOL Get_field_7CF() const { return field_7CF; }
     BOOL Get_field_7D3() const { return field_7D3; }
 
+    BOOL Evacuate_Civillians() { return CivEvac; }
```

Comment:
Same typo here. Function should also be const and there is no need for the whitespace between this and the Get_field_x above either.